### PR TITLE
Final commit

### PR DESCRIPTION
**Pull request by yunko1803**
Fix all the last bugs

### CODE REVIEW

**Review by Jen-Lopez — APPROVED** [review]
wonderful

**Comment by Jen-Lopez** [comment]
Actually, there's this weird bug. Been testing it for 2 users, and it seems like if both users choose different cuisines,  it'll update the db 1+ times and display different winning cuisines. 

db
backend
frontend

File: back-end/routes/win.js [hunk 1]
```diff
@@ -32,12 +30,13 @@ router.get("/win", function (req, res) {
         else {
             finalCuisine = selectedCuisinesList[parseInt(Math.random() * selectedCuisinesList.length)]
         }
+
```
Comment (Jen-Lopez) [inline back-end/routes/win.js:33]: before figuring out the winning cuisine, I would check if `finalCuisine` is "". If it is, then we can generate the cuisine (at least in the case where there's a tie).

**Review by thomastai1666 — APPROVED** [review]

**Review by Golemwardox — APPROVED** [review]

File: back-end/routes/win.js (hunk 1 above)
Comment (yunko1803) [reply back-end/routes/win.js:33]: i see

**Pull request merged**